charts/ingress-nginx/values.yaml

```diff
@@ -642,6 +642,8 @@ defaultBackend:
 rbac:
   create: true
   scope: false
+  extraRoleRules: []
```

Comment:
Please add some comments about the settings

Reply:
added a few comments